packages/forms-web-app/__tests__/unit/routes/householder-planning/index.test.js

```diff
@@ -1,14 +1,16 @@
 const { use } = require('../router-mock');
 
-describe('routes/householder-planning', () => {
+const eligibilityRouter = require('../../../../src/routes/householder-planning/index');
+
+describe('routes/householder-planning/index', () => {
   beforeEach(() => {
     jest.resetModules();
 
     // eslint-disable-next-line global-require
-    require('../../../../src/routes/householder-planning');
+    require('../../../../src/routes/householder-planning/index');
```

Comment:
This still works but shouldn't be necessary.

Reply:
Edited.